### ISSUE TITLE
Undefined variable: "$icon-delete-color"

### ISSUE THREAD

**Issue by Ornanovitch** [mwalbeck/nextcloud-breeze-dark#103]
Hey @mwalbeck and thanks you for your work !

I'd forked your repo to build a personalized dark theme for my nextcloud (just changing some colors), but when I run `sass server.scss` in css/, I got the following message : 

```
Error: Undefined variable: "$icon-delete-color".
        on line 39 of apps/thirdparty/_quicknotes.scss
        from line 56 of server.scss
  Use --trace for backtrace.
```
What should I do ?

Thank you very much

**Comment by mwalbeck** [comment]
Hi,

That looks like a little bug. I have more or less moved away from sass variables, and I guess i missed one. Apparently the Nextcloud sass compiler is also a lot more lenient when it comes to undefined variables, since it has work for a couple of months now.

Enough about that, you can change out this line in the _quicknotes.scss file:
```
background-image: $icon-delete-color;
```
with
```
background-image: var(--color-error);
```

Then it should work as intended. As a little side note, you just have to place your custom theme in the Nextcloud app folder without compiling any sass, then Nextcloud will handle the rest.

**Comment by mwalbeck** [comment]
A fix for this has now also been pushed to master

**Comment by Ornanovitch** [comment]
I pulled your commit, it's working like a charm, thank you :)

Great work !

**Comment by mwalbeck** [comment]
No problem. The fix is now included with 18.0.10

Cheers!
